src/main/java/dan200/computercraft/shared/peripheral/diskdrive/DiskDrivePeripheral.java

```diff
@@ -76,18 +99,35 @@ public final void setDiskLabel( Optional<String> labelA ) throws LuaException
         diskDrive.setDiskStack( stack );
     }
 
+    /**
+     * Returns whether a disk with data is inserted.
+     *
+     * @param computer The computer object
+     * @return Whether a disk with data is inserted.
+     */
     @LuaFunction
     public final boolean hasData( IComputerAccess computer )
     {
         return diskDrive.getDiskMountPath( computer ) != null;
     }
 
+    /**
+     * Returns the mount path for the inserted disk.
+     *
+     * @param computer The computer object
+     * @return The mount path for the disk, or {@code nil} if no data disk is inserted.
+     */
     @LuaFunction
     public final String getMountPath( IComputerAccess computer )
```

Comment:
```suggestion
    @Nullable
    public final String getMountPath( IComputerAccess computer )
```

Can you chuck a `@Nullable` here? Should mean the inferred return type is correct.